metpy/calc/tools.py

```diff
@@ -157,3 +158,35 @@ def interpolate_nans(x, y, kind='linear'):
     else:
         raise ValueError('Unknown option for kind: {0}'.format(str(kind)))
     return y[x_sort_args]
+
+
+@exporter.export
```

Comment:
Given that this could go away in the future, I'm not ready to export this. I'd even consider naming with a leading underscore.